src/context/SyntheticsStateContext/selectors/tradeboxSelectors/index.ts

```diff
@@ -348,6 +352,23 @@ const selectTradeboxEstimatedGas = createSelector(function selectTradeboxEstimat
   }
 });
 
+const selectTradeboxSwapCount = createSelector(function selectTradeboxSwapCount(q) {
+  const { isSwap, isIncrease } = q(selectTradeboxTradeFlags);
+  if (isSwap) {
+    const swapAmounts = q(selectTradeboxSwapAmounts);
+    if (!swapAmounts) return undefined;
+    return swapAmounts?.swapPathStats?.swapPath.length ?? 0;
```

Comment:
i guess `?` in `swapAmounts?` is not needed because it's already checked on the previous line?

Reply:
fixed